### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by andrewmathies**

### CODE REVIEW

**Comment by andrewmathies** [comment]
I tried to JSdoc all of the code services you marked, as well as update the package.json file to match https://github.com/rreinold/twilio-sms-library/blob/master and other ipm packag.json files. Also updated readme after checking the best practices for IPM and readme documents. 

I think there still are things that can be improved, so I will continue working on it

**Comment by andrewmathies** [comment]
@yashjain28 I forgot to ping you in the above message, I'm still working on it but I have made some changes

File: code/services/GetRecentAnomalies/GetRecentAnomalies.js [hunk 1]
```diff
@@ -1,7 +1,20 @@
 /**
  * This service retrieves the 10 most recent data points from the Anomalies collection.
  * 
- * @returns {Object[]} data
+ * @typedef {timestamp} self-explanatory type
```
Comment (yashjain28) [inline code/services/GetRecentAnomalies/GetRecentAnomalies.js]: Refer jsdoc.org they usually put @typedef in a seperate block /** */
@typedef {timestamp} self-explanatory type => not quite sure what is being implied here.

File: code/services/SendAlert/SendAlert.js [hunk 2]
```diff
@@ -90,16 +95,17 @@ function SendAlert(req, resp){
             log("Successfully sent SMS")
         }
     }
+
+    /** @global */
+    var template = "Sensor {{sensor_key}} breached threshold by {{variancePercent}}% with value of {{val}}";
```
Comment (yashjain28) [inline code/services/SendAlert/SendAlert.js]: I am not sure if you understood what I meant by making this statement editable globally. I'll rephrase what I meant: The goal was to make it resuable, by putting it in the contants library. 

But it wouldn't be required as the data in the anomaly object is coming from a collections, so attributes are fixed. Hence, not much resuability. So I'll say put it back inside the function scope. (as originally)

File: README.md [hunk 3]
```diff
@@ -29,22 +34,23 @@ To see a visualization of the data, go to the Portals tab and click on the Anoma
...
 * NewThunderboardDevice
 * SendAlertsTrigger
 
-# Portals
+## Portals
 * AnomalyDetection
 
-# Thank You
-Powered by ClearBlade Enterprise IoT Platform: https://platform.clearblade.com/## Members
+## Thank You
```
[the remark this answers is not in the document]
Comment (andrewmathies) [reply README.md]: webhook adapter still has thank you section @yashjain28

**Pull request merged**